reference/5.1/Microsoft.PowerShell.Management/Set-Location.md

```diff
@@ -69,27 +69,37 @@ Env:\
 PS Env:\>
 ```
 
-This command sets the current location to the root of the Env: drive. It uses the **PassThru**
-parameter to direct PowerShell to return a **PathInfo** object that represents the Env: location.
+This command sets the current location to the root of the `Env:` drive. It uses the **PassThru**
+parameter to direct PowerShell to return a **PathInfo** object that represents the `Env:\` location.
 
 ### Example 3: Set location to the C: drive
 
 ```powershell
-PS C:\> Set-Location C:
+PS Env:\> Set-Location C:\
 ```
 
-This command sets the current location C: drive in the FileSystem provider.
+This command sets the current location to the root of the `C:` drive in the FileSystem provider.
```

Comment:
Do you think we should change this one? It highlights that setting the location without specifically defining the location will push the location to the current location in the FileSystem provider.  I think from the separate provider change is good though so something like:

`PS Env:\> Set-Location -Path C:`

Reply:
I get the point.
Maybe like this 91812d0d4863a5ab8e4fbbffb899299c504378ad?